webcompat/api/endpoints.py

```diff
@@ -214,4 +217,4 @@ def get_repo_labels():
     """XHR endpoint to get all possible labels in a repo."""
     params = request.args.copy()
     path = 'repos/{0}/labels'.format(REPO_PATH)
-    return api_request('get', path, params=params)
+    return api_request('get', path, params=params, mime_type=JSON_MIME_HTML)
```

Comment:
this one doesn't seem to be about issues. ;)

Reply:
This was wrong. I missed it.
Thanks Karl!